core/src/main/scala/cats/Reducible.scala

```diff
@@ -112,6 +112,12 @@ import simulacrum.typeclass
       val F = self
       val G = Reducible[G]
     }
+
+  def minimum[A](fa: F[A])(implicit A: Order[A]): A =
```

Comment:
I see...

I'm not a huge implicit syntax fan (this example, but also confusion for novices as to where methods are coming from).

But I recognize many cats users do like this, so avoiding the collision seems safest.

👍